src/Expand/Expand.js

```diff
@@ -0,0 +1,86 @@
+import React, { Component, PropTypes } from 'react'
+import { View, Text, StyleSheet, TouchableOpacity } from 'react-native'
+import Icon from 'react-native-vector-icons/FontAwesome'
+import themeable from '../utils/themeable'
+
+class Expand extends Component {
+
+  static propTypes = {
+    title: PropTypes.string.isRequired,
+    description: PropTypes.string,
+    children: PropTypes.node,
+    defaultExpanded: PropTypes.bool,
+    styles: PropTypes.object,
+  }
+
+  static defaultProps = {
+    description: '',
+    children: null,
+    defaultExpanded: false,
+    styles: null,
+  }
+
+  state = { expanded: this.props.defaultExpanded }
+
+  onPress = () => {
+    this.setState({ expanded: !this.state.expanded })
+  }
+
+  render() {
+    const { title, description, children, styles } = this.props
+    const { expanded } = this.state
+    const iconName = expanded ? 'chevron-up' : 'chevron-down'
+
+    return (
+      <TouchableOpacity onPress={this.onPress} activeOpacity={1}>
+        <View style={styles.container}>
+          <View style={styles.titleContainer}>
+            <Text numberOfLines={1}>
+              {title}
+            </Text>
+            <Icon
+              name={iconName}
+              color="#9FADBA"
```

Comment:
I think this property should be customizable from the outside and should have default value from ThemeConstants // @itsmepetrov